migrate/phases/5-edx.txt

```diff
@@ -1,5 +1 @@
-edx-ora2
-edx-proctoring
-edx-submissions
-docs.edx.org
```

Comment:
Where did `docs.edx.org` get moved to?

Reply:
We moved it to Phase 8. I forgot to add the phase 8 txt file for it.  I've added it now.